docs/user_guide/assessment.rst

```diff
@@ -396,15 +396,29 @@ For more examples, please
 see the :ref:`sphx_glr_auto_examples_plot_new_metrics.py` notebook in the
 :ref:`examples`.
 
+.. _plot:
+
+Plotting grouped metrics
+^^^^^^^^^^^^^^^^^^^^^^^^
+
+The simplest way to visualize grouped metrics from the :class:`MetricFrame` is
+to take advantage of the inherent plotting capabilities of
+:class:`pandas.DataFrame`:
+
+.. literalinclude:: ../auto_examples/plot_quickstart.py
+    :lines: 26-37
```

Comment:
See comment above about possible changes to `literalinclude`

Reply:
Done!